docs/README.rst

```diff
@@ -52,7 +52,9 @@ credentials files.
 
 .. code:: bash
 
-    source .venv/bin/activate # activate the `venv` virtual environment
+    # activate the `.venv` virtual environment
+    source .venv/bin/activate   # on Linux and MacOS
+    .\.venv\Scripts\activate    # on Windows
```

Comment:
@lopierra: I added these lines to address issue #482. Do you think this is sufficient? I think most of the other commands work the same way on Windows as they do in a Unix based environment, right?